core/src/main/java/dk/alexandra/fresco/lib/real/DefaultLinearAlgebra.java

```diff
@@ -73,6 +73,30 @@ protected DefaultLinearAlgebra(ProtocolBuilderNumeric builder) {
     });
   }
 
+  @Override
+  public DRes<Matrix<DRes<SReal>>> sub(DRes<Matrix<DRes<SReal>>> a, DRes<Matrix<DRes<SReal>>> b) {
+    return builder.par(par -> {
+      return add(par, a.out(), b.out(),
```

Comment:
Please find a better name for the `add` method used here. The fact, that it is used to subtract here, suggests that the name is inaccurate :).